spec/unit/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,51 @@
+RSpec.describe ApplicationHelper, type: :view do
```

Comment:
I actually think probably no type is okay here, but what I've done is made them helper tests by removing all the explicit test types and just rearranging into directories so that `config.infer_spec_type_from_file_location!` works (I've done that for all types - system and requests were already like this, but it means moving helpers and models out into their own directories).